### DISCUSSION TITLE
Styling of tooltips

### DISCUSSION THREAD

**Discussion by MartyIX** [dotnet/maui#20804] [Q&A] [answered]
I have read https://learn.microsoft.com/en-gb/dotnet/maui/user-interface/tooltips?view=net-maui-8.0 but it's not exactly clear to me whether one can change font family/font size/font color/etc. for the tooltip or not. Is it supported or not?

Thank you

**Comment by ghost** [comment, marked as answer]
CoPilot [found this solution for Windows](https://learn.microsoft.com/en-us/answers/questions/1605395/how-to-customize-maui-tooltips-for-windows), but could not find alternative for Android/iOS. **Easily**? No. **Workaround**? Possible, did not found it with quick search.
